var/spack/repos/builtin/packages/py-protobuf/package.py

```diff
@@ -85,7 +70,7 @@ class PyProtobuf(PythonPackage):
     for ver in list(range(21, 22)):
         depends_on("protobuf@3." + str(ver), when="+cpp @4." + str(ver))
     # Handle the 3.x series releases
-    for ver in list(range(1, 8)) + list(range(9, 21)):
```

Comment:
Why include `1`, `2`, `13`, and `14` when they aren't in the version's list for `py-protobuf`?

Reply:
Just to simplify the for-loop.